LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2024 Rocky Mountain Institute
```

Comment:
It's just `RMI` now, I believe (right @hodie?)

Reply:
TIL: I don't know what company I work for